spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/batch/SimpleJobServiceFactoryBean.java

```diff
@@ -83,10 +88,24 @@ public class SimpleJobServiceFactoryBean implements FactoryBean<JobService>, Ini
 
 	private PlatformTransactionManager transactionManager;
 
+	private JobServiceContainer jobServiceContainer;
+
+	private SchemaService schemaService;
+
+	private AppBootSchemaVersion appBootSchemaVersion;
```

Comment:
The SchemaTarget is better since it may contain different prefixes in the case where we have multiple databases or targets per database.